src/components/domain/article/ArticleBody/ArticleBody.tsx

```
@@ -0,0 +1,260 @@
+ * - Images: Zoom on hover (scale 1.1), 0.3s ease, cursor zoom-in
+ * - Transition: background-size 0.4s ease
+ */
+export const ArticleBody = ({ content, className }: ArticleBodyProps) => {
+  // Convert relative Drupal image URLs to absolute URLs
+  const processedContent = convertDrupalImagesToAbsolute(content)
+
+  return (
+    <>
+      {/* Inline styles for article body - using dangerouslySetInnerHTML for Storybook compatibility */}
+      <style
+        dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)